source/release-notes/1.4.txt

```diff
@@ -0,0 +1,33 @@
+.. _c2c-release-notes-1.4:
+
+===================================
+Release Notes for ``mongosync`` 1.4
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. note::
+
+   ``mongosync`` 1.4 Released <<TBD>>, 2023
+
+New Features
+-------------
+
+Set Cluster Workload Level
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in 1.4.0, {+c2c-product-name+} allows you to set the cluster
+workload level used for synchronizing data between the source and
```

Comment:
As above for syncing - synchronizing

Consider making more active:
Starting in 1.4.0, you can set the workload level that {+c2c-product-name+} uses to sync data between the source and destination clusters.

Reply:
Done.